man/systemd.network.xml

```diff
@@ -1281,6 +1281,12 @@ Table=1234</programlisting></para>
             unset.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>SuppressIfGroup=</varname></term>
```

Comment:
I'm not crazy about the option name. Something more self-explanatory would be nice, but I don't have a good idea.

Please explain what this option does here, and what the valid values are.

A good start is the kernel commit message:
   This change adds the ability to suppress a routing decision based upon the
    interface group the selected interface belongs to. This allows it to
    exclude specific devices from a routing decision.

Reply:
Hey, sorry for this I was going to update it on a previous weekend but didn't had enough time so it was left like this (that's partially why I named it as a `WIP`, to prevent from merging). I published it just because wanted to know opinion if I could change `Group` type from a `uint32_t` to `int32_t`.